### PR TITLE
Martiora

### PR DESCRIPTION
**Pull request by Tanosimboangy**
Hey Shawn 

Could check this one please.

Thanks

jacquit

### CODE REVIEW

File: m/martiora.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Martiora Freedom
+
+![Image of martiora freedom](martiora-freedom.jpg)
+
+Martiora Freedom is a malagasy group, which was created in 2009 in Antananarivo. This is formed by five members : Tobi, Malm, Bomi, Tremai and Mafr (entered in 2013). This group started to sing in 2009 and became famous in 2012 with the first song << Hafa mihintsy >>. They released their first album in February 2013. On 7th November 2014, << Best Of Timoramar >> was released, their second album. Since 2015, this group reached their success untill now.
```
Comment (ndorfin) [inline m/martiora.md]: Please fix the formatting in this bio. e.g. Use double quotes rather than `<<`

File: m/martiora.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Martiora Freedom
```
Comment (ndorfin) [inline m/martiora.md:1]: The filename for this artist looks wrong. Shouldn't it be `martiora-freedom`?

**Review by ndorfin — CHANGES_REQUESTED** [review]
The extra JPG can be removed too.

File: m/martiora.md (hunk 1 above)
Comment (Tanosimboangy) [reply m/martiora.md]: I'm sorry Shawn , I don't understand the JPG. Could you explain please?

File: m/martiora.md (hunk 2 above)
Comment (Tanosimboangy) [reply m/martiora.md:1]: This is the new one, please could you check?

**Pull request closed without merging**